packages/react-storage/src/components/StorageBrowser/views/LocationActionView/__tests__/UploadControls.spec.tsx

```diff
@@ -1,162 +1,59 @@
+}));
+
+jest.spyOn(ConfigModule, 'useGetActionInput').mockReturnValue(config);
 
 describe('UploadControls', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
-  it('should render upload controls table', async () => {
-    await waitFor(() => {
-      render(
-        <Provider>
-          <UploadControls />
-        </Provider>
-      );
-    });
+  it('should render upload controls table', () => {
+    const { getByRole } = render(<UploadControls />);
 
-    const table = screen.getByRole('table');
+    const table = getByRole('table');
     expect(table).toBeInTheDocument();
   });
 
-  it('should render the destination folder', async () => {
-    await waitFor(() => {
-      render(
-        <Provider>
-          <UploadControls />
-        </Provider>
-      );
-    });
+  it('should render the destination folder', () => {
+    const { getByText } = render(<UploadControls />);
 
-    const destination = screen.getByText('Destination:');
-    const destinationFolder = screen.getByText('folder3/');
+    const destination = getByText('Destination:');
+    const destinationFolder = getByText('test-prefix/');
 
     expect(destination).toBeInTheDocument();
     expect(destinationFolder).toBeInTheDocument();
   });
-
-  it('opens the file picker when the add files button is clicked and uses the file selection dialog', async () => {
-    const user = userEvent.setup();
-    const files = [
-      new File(['content1'], 'file1.txt', { type: 'text/plain' }),
-      new File(['content2'], 'file2.txt', { type: 'text/plain' }),
-      new File(['content3'], 'file3.txt', {
-        type: 'text/plain',
-      }),
-    ];
-
-    render(
-      <Provider>
-        <UploadControls />
-      </Provider>
-    );
-
-    const button = screen.getByRole('button', { name: 'Add files' });
-    const input: HTMLInputElement = screen.getByTestId('amplify-file-select');
-
-    expect(input).toHaveAttribute('multiple');
-
-    await act(async () => {
-      await user.click(button);
-      await user.upload(input, files);
-    });
-
-    expect(input.files).toHaveLength(3);
-  });
-
-  it('adds files dragged into the drop zone to the file list', async () => {
-    const files = [new File(['content'], 'file.txt', { type: 'text/plain' })];
-
-    render(
-      <Provider>
-        <UploadControls />
-      </Provider>
-    );
-
-    const dropzone = screen.getByTestId('storage-browser-table');
-
-    fireEvent.drop(dropzone, {
-      dataTransfer: {
-        files,
-      },
-    });
-
-    await waitFor(() => {
-      expect(screen.getByText('file.txt')).toBeInTheDocument();
-    });
-  });
-
-  it('has the webkitdirectory attribute for the input select for folders', () => {
-    render(
-      <Provider>
-        <UploadControls />
-      </Provider>
-    );
-
-    const input: HTMLInputElement = screen.getByTestId('amplify-folder-select');
-
-    expect(input).toHaveAttribute('webkitdirectory');
-  });
```

Comment:
Removed these tests as they were testing the logic of underlying components and will be covered by the upcoming `DropZone` composable